doc/reflection-milestone2.md

```diff
@@ -0,0 +1,61 @@
+## Milestone 2 Reflection: Foundation Shades Across the Globe
+
+### 1. What we have implemented
+
+We have implemented all the 3 sections we promised to implement from Milestone 1, which are:
```

Comment:
Wording change suggestion:

We have implement all 3 sections outlined in our Milestone 1 proposal, which include: